src/InstrumentationEngine/ProfilerManager.cpp

```diff
@@ -859,7 +862,7 @@ HRESULT MicrosoftInstrumentationEngine::CProfilerManager::SetEventMask2(_In_ DWO
     return S_OK;
 }
 
-MicrosoftInstrumentationEngine::ClrVersion MicrosoftInstrumentationEngine::CProfilerManager::GetAttachedClrVersion()
```

Comment:
>MicrosoftInstrumentationEngine [](start = 43, length = 30)

I can revert this... or update the entire file to remove this namespace since it makes this file harder to read.

Reply:
You can go either way as long as it is consistent. I prefer not having the namespace everywhere (your latter suggestion).

Reply:
If you are going to make this change, can you update the rest of the file as well?

---
In reply to: [331296901](https://github.com/microsoft/CLRInstrumentationEngine/pull/146#discussion_r331296901) [](ancestors = 331296901)